pasta_eln/gui.py

```diff
@@ -49,44 +49,43 @@ def __init__(self) -> None:
         if docType[0] == 'x' and docType[1] != '0':
           continue
         shortcut = f"Ctrl+{shortCuts[docType]}" if docType in shortCuts else None
-        Action(docLabel, self, [Command.VIEW, docType], viewMenu, shortcut=shortcut)
+        Action(docLabel,                     self, [Command.VIEW, docType],  viewMenu, shortcut=shortcut)
         if docType == 'x0':
           viewMenu.addSeparator()
       viewMenu.addSeparator()
-      Action('&Tags', self, [Command.VIEW, '_tags_'], viewMenu, shortcut='Ctrl+T')
-      Action('&Unidentified', self, [Command.VIEW, '-'], viewMenu, shortcut='Ctrl+U')
+      Action('&Tags',                        self, [Command.VIEW, '_tags_'], viewMenu, shortcut='Ctrl+T')
+      Action('&Unidentified',                self, [Command.VIEW, '-'],      viewMenu, shortcut='Ctrl+U')
 
     systemMenu = menu.addMenu("&System")
-    Action('&Project groups', self, [Command.PROJECT_GROUP], systemMenu)
+    Action('&Project groups',                self, [Command.PROJECT_GROUP],   systemMenu)
     changeProjectGroups = systemMenu.addMenu("&Change project group")
-    if hasattr(self.backend, 'configuration'):  # not case in fresh install
+    if hasattr(self.backend, 'configuration'):                            # not case in fresh install
       for name in self.backend.configuration['projectGroups'].keys():
-        Action(name, self, [Command.CHANGE_PG, name], changeProjectGroups)
-    Action('&Syncronize', self, [Command.SYNC], systemMenu, shortcut='F5')
-    Action('&Questionaires', self, [Command.ONTOLOGY], systemMenu, shortcut='F8')
+        Action(name,                         self, [Command.CHANGE_PG, name], changeProjectGroups)
+    Action('&Syncronize',                    self, [Command.SYNC],            systemMenu, shortcut='F5')
+    Action('&Questionaires',                 self, [Command.ONTOLOGY],        systemMenu, shortcut='F8')
     systemMenu.addSeparator()
-    Action('&Test extraction from a file', self, [Command.TEST1], systemMenu)
-    Action('Test &selected item extraction', self, [Command.TEST2], systemMenu, shortcut='F2')
-    Action('Update &Extractor list', self, [Command.UPDATE], systemMenu)
+    Action('&Test extraction from a file',   self, [Command.TEST1],           systemMenu)
+    Action('Test &selected item extraction', self, [Command.TEST2],           systemMenu, shortcut='F2')
+    Action('Update &Extractor list',         self, [Command.UPDATE],          systemMenu)
     systemMenu.addSeparator()
-    Action('&Configuration', self, [Command.CONFIG], systemMenu, shortcut='Ctrl+0')
+    Action('&Configuration',                 self, [Command.CONFIG],          systemMenu, shortcut='Ctrl+0')
 
     helpMenu = menu.addMenu("&Help")
-    Action('&Website', self, [Command.WEBSITE], helpMenu)
-    Action('&Verify database', self, [Command.VERIFY_DB], helpMenu, shortcut='Ctrl+?')
-    Action('Shortcuts', self, [Command.SHORTCUTS], helpMenu)
-    helpMenu.addSeparator()
```

Comment:
Is this separator removed intentionally?

Reply:
Yes, it was intentionally. A separator at the end of a menu is meaningless.